pkg/hubble/filters/cel_expression.go

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Hubble
+
+package filters
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	flowpb "github.com/cilium/cilium/api/v1/flow"
+	v1 "github.com/cilium/cilium/pkg/hubble/api/v1"
+	"github.com/cilium/cilium/pkg/logging"
+	"github.com/cilium/cilium/pkg/logging/logfields"
+
+	"github.com/google/cel-go/cel"
+)
+
+var (
+	celFilterLogger = logging.DefaultLogger.WithField(logfields.LogSubsys, "cel-filter")
```

Comment:
I think we should use the `hubble` logger here (`ie `subsys-hubble`).

Reply:
I'll give it a try, but unfortunately I think doing that is going to require a fairly decent refactor to everything using the filters subsystem to support it, since we create a bunch of global singletons. It should be fairly straight through to plumb it down the stack though.

Reply:
Pushed as a new commit. PTAL.